plugins/pulp_rpm/plugins/db/models.py

```diff
@@ -553,6 +554,17 @@ def download_path(self):
         """
         return self.filename
 
+    def verify_size(self, location):
+        """
+        Verify size of the file at the specified location
+
+        :param location: path to the unit location
+        :type  location: str
+        """
+        if self.size is not None:
+            with open(location) as fp:
+                verification.verify_size(fp, self.size)
```

Comment:
This function is used for the size verification in several places including the verification of rpm right after download and before copy (and in this PR it is after-copy). So to be consistent, maybe it is better either to modify the `verification.verify_size` function itself or to use it as is. 

What do you think? (And I am trying to make less changes to Pulp 2 code if they are not necessary, but maybe it is not a right approach)